<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="lay_prin">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="carte">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="interactive">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
      <property name="cacheMode">
       <set>QGraphicsView::CacheNone</set>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::MinimalViewportUpdate</enum>
      </property>
      <property name="rubberBandSelectionMode">
       <enum>Qt::ContainsItemShape</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="texte">
      <property name="text">
       <string>pas d'info</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_fichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="menu_fichier_enregistrer"/>
    <addaction name="menu_fichier_charger"/>
    <addaction name="menu_fichier_nouvelle"/>
    <addaction name="menu_fichier_quitter"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionChanger_la_valeure_du_zoom"/>
   </widget>
   <widget class="QMenu" name="menuClaque">
    <property name="title">
     <string>Claque</string>
    </property>
    <addaction name="actionCalque_1"/>
    <addaction name="actionCalque_2"/>
    <addaction name="actionCalque_3"/>
   </widget>
   <widget class="QMenu" name="menuOutil">
    <property name="title">
     <string>Outil</string>
    </property>
    <addaction name="actionObjet"/>
    <addaction name="actionCase_marchable"/>
    <addaction name="actionCase_fight"/>
    <addaction name="actionTransporteur"/>
    <addaction name="actionCase_portee"/>
   </widget>
   <widget class="QMenu" name="menuCase_fight">
    <property name="title">
     <string>Case fight</string>
    </property>
    <addaction name="actionEquipe_1"/>
    <addaction name="actionEquipe_2"/>
   </widget>
   <widget class="QMenu" name="menuCase_marchable">
    <property name="title">
     <string>Case marchable</string>
    </property>
    <addaction name="actionPetite"/>
    <addaction name="actionMoyenne"/>
    <addaction name="actionGrande"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
   </widget>
   <addaction name="menu_fichier"/>
   <addaction name="menuZoom"/>
   <addaction name="menuClaque"/>
   <addaction name="menuOutil"/>
   <addaction name="menuCase_fight"/>
   <addaction name="menuCase_marchable"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QToolBar" name="barre_outil">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="menu_fichier_enregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="menu_fichier_quitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="menu_objet_nouveau">
   <property name="text">
    <string>nouveau</string>
   </property>
  </action>
  <action name="menu_fichier_nouvelle">
   <property name="text">
    <string>Nouvelle</string>
   </property>
  </action>
  <action name="menu_fichier_charger">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
  <action name="menu_fond_nouveau">
   <property name="text">
    <string>nouveau</string>
   </property>
  </action>
  <action name="menu_fond_remplire">
   <property name="text">
    <string>remplire</string>
   </property>
  </action>
  <action name="menu_fond_charger">
   <property name="text">
    <string>Je veux plus de fond</string>
   </property>
  </action>
  <action name="actionChanger_la_valeure_du_zoom">
   <property name="text">
    <string>Changer la valeure du zoom</string>
   </property>
  </action>
  <action name="actionCalque_1">
   <property name="text">
    <string>Calque 1</string>
   </property>
  </action>
  <action name="actionCalque_2">
   <property name="text">
    <string>Calque 2</string>
   </property>
  </action>
  <action name="actionCalque_3">
   <property name="text">
    <string>Calque 3</string>
   </property>
  </action>
  <action name="actionObjet">
   <property name="text">
    <string>Objet</string>
   </property>
  </action>
  <action name="actionCase_marchable">
   <property name="text">
    <string>Case marchable</string>
   </property>
  </action>
  <action name="actionCase_fight">
   <property name="text">
    <string>Case fight</string>
   </property>
  </action>
  <action name="actionTransporteur">
   <property name="text">
    <string>Transporteur</string>
   </property>
  </action>
  <action name="actionEquipe_1">
   <property name="text">
    <string>Equipe 1</string>
   </property>
  </action>
  <action name="actionEquipe_2">
   <property name="text">
    <string>Equipe 2</string>
   </property>
  </action>
  <action name="actionPetite">
   <property name="text">
    <string>Petite</string>
   </property>
  </action>
  <action name="actionMoyenne">
   <property name="text">
    <string>Moyenne</string>
   </property>
  </action>
  <action name="actionGrande">
   <property name="text">
    <string>Grande</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
  </action>
  <action name="actionCase_portee">
   <property name="text">
    <string>Case port√©e</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
